### ISSUE TITLE
[FAQ] Add FAQ about triggering bindings with non-English keyboards

### ISSUE THREAD

**Issue by rodrigogiraoserrao** [Textualize/textual#3694]
See #3693 for the motivation behind this FAQ entry.
As more non-English users adopt Textual, it'll become increasingly important to make accessible the information that explains how bindings will seem to interact poorly with keyboards that are set to languages with alphabets other than the latin one.

**Comment by willmcgugan** [comment]
@rodrigogiraoserrao Can you summarize what the user needs to know here, for my benefit?

**Comment by rodrigogiraoserrao** [comment]
You're deep in Textual, so I can summarise it as “the key that Textual recognises may not match the symbol shown in the physical key in the physical keyboard you pressed”.

Why I think this warrants a FAQ entry despite being “so simple”:

Someone new to Textua may be caught off guard when using international keyboards.
E.g., the video below shows that by switching between PT or US keyboard layouts, the same physical key on my keyboard will be recognised differently by Textual.
And PT <> US doesn't have that many differences...

Imagine if I set my keyboard to an Arabic mode, or Chinese, or Japanese, or ...
Then, none of the keys I press will match the typical Textual bindings.
In hindsight, it's “obvious” but I just want to make life easier for folks using keyboards in other modes.

https://github.com/Textualize/textual/assets/5621605/756ca61a-4fb7-4a75-b4da-796ef70484b8

**Comment by willmcgugan** [comment]
Not sure it is much of a problem in practice. Will wait until we're asked about it.
